src/cryptoadvance/specter/util/common.py

```diff
@@ -34,7 +35,9 @@ def replace_substring(text, start_position, replace_length, new_str):
 
 
 def satamount_formatted(value):
-    return "{:,.0f}".format(round(float(value) * 1e8))
+    # TODO: amounts should be handled internally only as integer sats; no floats or str,
+    # except for when formatting for display.
+    return "{:,d}".format(int(Decimal(value)*Decimal(1e8)))
```

Comment:
Just curious, is the "d" actually doing anything here or is more to show that we are converting an integer to a string?

Reply:
Interesting. Didn't realize it would just do the right thing for ints without the `d` specifier. I guess it's a holdover habit on my part from the bad old `"blah %d foo %s" % (some_int, some_str)` string formatting days.

Still, I prefer everything to be as explicit as possible.